cypress/e2e/jackwang.cy.ts

```diff
@@ -86,5 +86,47 @@ it('33678', () =>{
     cy.contains("Good job!")
 })
 
+//***************************************************************************
+it('23292', () =>{
+    cy.visit('https://obstaclecourse.tricentis.com/Obstacles/23292')
+    let taskIDs = []
+    cy.get('tbody tr.draggable-row').each(($row)=>{
+        const id = parseInt($row.find('td').eq(0).text(), 10)
+        taskIDs.push(id)
+    }).then(() => {
+        taskIDs.sort((a, b) => a - b)
+
+        for(let i = 0; i<taskIDs.length; i++){
```

Comment:
Can you use the dragTo method introduced by @kaka5277 ?
https://github.com/jun1983/cypress_training/blob/main/cypress/e2e/michael.cy.ts#L18